app/models/spree/tax_rate_decorator.rb

```diff
@@ -30,11 +24,6 @@ def adjust(order, item)
     raise SpreeAvatax::TaxRateInvalidOperation.new("Spree::TaxRate#adjust should never be called when Avatax is present")
   end
 
-  def compute_amount(item)
-    # Avatax tax adjustments should always be finalized so Spree should never attempt to call this code
-    raise SpreeAvatax::TaxRateInvalidOperation.new("Spree::TaxRate#compute_amount should never be called when Avatax is present")
-  end
```

Comment:
@mamhoff see the commit comments in https://github.com/solidusio/solidus_avatax/pull/19/commits/cc6cb8103c6256cc47d5369fc76988646bd1e310 -- this ends up calling `SpreeAvatax::Calculator#compute` which already has a similar safeguard in place. So this shouldn't be necessary and we should still be covered.  solidus_avatax doesn't yet handle VAT or calculating taxes on shipping rates so the shipping rate logic should be fine for now.